### DISCUSSION TITLE
How do I disable webworkers when using file://

### DISCUSSION THREAD

**Discussion by chaoticryptidz** [PrismJS/prism#2855] [Q&A]
Prism gives me these errors when trying to use it via file:///

```
prism.js:3 Uncaught DOMException: Failed to construct 'Worker': Script at 'file:///home/username/Projects/test/js/prism.js' cannot be accessed from origin 'null'.
    at highlightElement (file:///home/username/Projects/test/js/prism.js:3:3523)
```

**Comment by RunDevelopment** [comment]
WebWorkers are disabled by default. They can be enabled by using the [`async` parameter](https://prismjs.com/docs/Prism.html#.highlightElement).

Are you calling any of the `Prism.highlight*` functions yourself?

**Reply by chaoticryptidz** [reply]
Yes I am using highlightElement with codejar editor.

**Reply by RunDevelopment** [reply]
Well, is the second parameter when calling the function `true`?
